databuilder/extractor/dashboard/mode_analytics/mode_dashboard_last_successful_executions_extractor.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from pyhocon import ConfigTree, ConfigFactory  # noqa: F401
+
+from databuilder.extractor.dashboard.mode_analytics.mode_dashboard_executions_extractor import \
+    ModeDashboardExecutionsExtractor
+from databuilder.extractor.dashboard.mode_analytics.mode_dashboard_utils import ModeDashboardUtils
+from databuilder.extractor.restapi.rest_api_extractor import STATIC_RECORD_DICT
+from databuilder.models.dashboard.dashboard_execution import DashboardExecution
+from databuilder.rest_api.rest_api_query import RestApiQuery
+
+LOGGER = logging.getLogger(__name__)
+
+
+class ModeDashboardLastSuccessfulExecutionExtractor(ModeDashboardExecutionsExtractor):
+    """
+    A Extractor that extracts Mode dashboard's last successful run (execution) timestamp.
+
+    """
+
+    def __init__(self):
+        super(ModeDashboardLastSuccessfulExecutionExtractor, self).__init__()
```

Comment:
if you don't put this line, wouldn't it auto init use the parent class constructer?

Reply:
> if you don't put this line, wouldn't it auto init use the parent class constructer?

Mmm.. I checked in multiple places but I haven't seen auto init..

https://stackoverflow.com/questions/1607612/python-how-do-i-make-a-subclass-from-a-superclass